LANGS.md

```diff
@@ -1 +1,2 @@
-* [English](en/)
\ No newline at end of file
+* [English](en/)
+* [Français](fr/)
```

Comment:
Just because I had written the contents in French… Perhaps this would be better put in a wiki.